README.md

```diff
@@ -89,7 +89,7 @@ const team = new service.Team("team", {
     displayName: "PulumiUP Team",
     description: "This was created with Pulumi",
     organizationName: "pulumi-up",
-    type: "pulumi",
+    teamType: "pulumi",
```

Comment:
Didn't mean to include this (and the one under `sdk/python/README.md`)... but doesn't hurt to merge it in